scripts/githooks/pre-commit

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+filter_formattable_files() {
```

Comment:
Add some high-level docs to this function. Bash commands aren't easy to understand